### PR TITLE
[#13004] Add Timestamp to Tag list API

### PR DESCRIPTION
**Pull request by CuriousFurBytes**
# Description

This Pull Request aims to solve the problem described on #13004, adding a timestamp to the tag list API.
As this is my first time writing Go code, please let me know if there's anything that could be improved.
I did not find any unit tests for the changed object.

## Results

When running the API locally, the swagger docs result in this:

Screen Shot 2020-10-03 at 14 34 05

And the API returns the following information:

Screen Shot 2020-10-03 at 14 34 21

### CODE REVIEW

**Comment by 6543** [comment]
Pleace add a test :)

**Comment by lucasqueiroz** [comment]
@6543 I've added unit tests for the `ToCommitMeta` method, not sure if the approach I took is the best, but I couldn't find any way to load Tags from a loaded bean.
Please let me know if anything can be improved

**Review by a1012112796 — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

File: modules/structs/repo_commit.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ type Identity struct {
 type CommitMeta struct {
 	URL string `json:"url"`
 	SHA string `json:"sha"`
+	// swagger:strfmt date-time
+	Timestamp time.Time `json:"timestamp"`
```
Comment (6543) [inline modules/structs/repo_commit.go]: one thing: the name

should we realy call it "timestamp" and not "created" ?

Comment (CuriousFurBytes) [reply modules/structs/repo_commit.go]: Looking at the GitHub API specifications, they usually use "date" as the field for dates, but to me created makes more sense. I'll change it!

**Comment by 6543** [comment]
@lucasqueiroz just `make fmt` :)

**Comment by lucasqueiroz** [comment]
> @lucasqueiroz just `make fmt` :)

Whoops my bad!

**Review by 6543 — APPROVED** [review]

**Pull request merged**